contracts/liquidity_hub/whale_lair/src/queries.rs

```diff
@@ -167,38 +188,24 @@ pub(crate) fn query_weight(
         total_bond_weight = total_bond_weight.checked_add(bond.weight)?;
     }
 
-    // for each of the unique denoms, access the unbondings
-    // and reduce the weight of the unbonding to the total_bond_weight
-    // TODO: REVIEW THIS WITH J
-    for denom in unique_denoms {
```

Comment:
This was not needed, as we already had calculated the weight for the decreased bonding before.